### ISSUE TITLE
Invite Tracking

### ISSUE THREAD

**Issue by iZeaoGamer** [discord-php/DiscordPHP#742]
So I've seen discord bots on discord that have an Invite tracking system, where you can track other users invites, so when they join the discord, it'll say smth like "They were invited by $inviter". My question is, is there a way to do this, or is it even possible using discord php? I've tried searching everywhere, but they result with a different coding language (I only know PHP Currently). I am sorry to disturb you, and I hope you can get back to me soon.

**Comment by key2peace** [comment]
no, because discord doesn't tell you which invite code they used to get in, it's difficult, especially when there is no use limit on the invite itself, if there is a use limit, you would need to keep track of the use count of each invite and attempt to determine it from there.

**Comment by iZeaoGamer** [comment]
Alright, thanks anyways!
